### PR TITLE
feature: TRACEFOSS-1111 update spring profiles on e2e env.

### PR DESCRIPTION
**Pull request by ds-mwesener**

### CODE REVIEW

**Comment by ds-mwesener** [comment]
Hi please ignore the frontend changes - I rebased and therefore there will be shown as changed but this is not correct. If you check diff from main you will not see those changes. Seems to be a github ui bug.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,17 +5,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased - x.x.x]
+### Added
+- Added option to hide "Investigation for components" in part-detail.component.html
+- E2E A env profile configuration
+- E2E B env profile configuration
```
Comment (ds-mmaul) [inline CHANGELOG.md:11]: "E2E Trace-X A and B profile configuration" - suggestion to summarize those two points into one

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,17 +5,30 @@ All notable changes to this project will be documented in this file.
...
+
+### Changed
+- Refactored other-parts.component.html - split into new components: supplier-parts.component.html and customer-parts.component.html
+- Bump cypress-io/github-action from 5.6.2 to 5.7.1
+- Bump surefire-plugin.version from 3.0.0-M8 to 3.0.0
+- Bump java-jwt from 4.3.0 to 4.4.0
+- Bump asciidoctor-maven-plugin from 2.2.2 to 2.2.3
+- Bump spring-cloud-dependencies from 2022.0.1 to 2022.0.2
+- Bump asciidoctorj from 2.5.7 to 2.5.8
```
Comment (ds-mmaul) [inline CHANGELOG.md:20]: Maybe summarize bumps?

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,17 +5,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased - x.x.x]
+### Added
+- Added option to hide "Investigation for components" in part-detail.component.html
+- E2E A env profile configuration
+- E2E B env profile configuration
+
+### Changed
+- Refactored other-parts.component.html - split into new components: supplier-parts.component.html and customer-parts.component.html
```
Comment (ds-mmaul) [inline CHANGELOG.md:14]: Less detail (exact file names) → explanation of the change

File: tx-backend/src/main/resources/application-e2e-a.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+################################################################################
+# Copyright (c) 2022, 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2022, 2023 ZF Friedrichshafen AG
+# Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
```
Comment (ds-mmaul) [inline tx-backend/src/main/resources/application-e2e-a.yml:4]: when new files get created --> only eclipse Copyright header + this year (delete 2022)

File: tx-backend/src/main/resources/application-e2e-b.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+################################################################################
+# Copyright (c) 2022, 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2022, 2023 ZF Friedrichshafen AG
+# Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
```
Comment (ds-mmaul) [inline tx-backend/src/main/resources/application-e2e-b.yml:4]: when new files get created --> only eclipse Copyright header + this year (delete 2022)

**Review by ds-mmaul — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (ds-mwesener) [reply CHANGELOG.md:14]: Hi Martin, this is already merged to main and to eclipse. Was not my part so I would like to not change it.

File: CHANGELOG.md (hunk 2 above)
Comment (ds-mwesener) [reply CHANGELOG.md:20]: I would like to have them line by line because every update is independend from each other.

File: CHANGELOG.md (hunk 1 above)
Comment (ds-mwesener) [reply CHANGELOG.md:11]: Addressed.

**Review by ds-mwesener — COMMENTED** [review]
Thanks everything has been adressed.

**Review by ds-mmaul — APPROVED** [review]
LGTM

**Pull request merged**